lib/offline.js

```diff
@@ -55,60 +61,121 @@ module.exports = function(config) {
       }
     }
 
-    resolve(ghslug('./').catch(function() {
+    ghslug('./')
+    .catch(function() {
       // Use the name from package.json if there's an error while fetching the GitHub slug.
       try {
         return JSON.parse(fs.readFileSync('package.json')).name;
       } catch (ex) {
         return '';
       }
-    }).then(function(cacheId) {
-      var staticFileGlobs = fileGlobs.map(function(v) {
+    })
+    .then(function(cacheId) {
+      checkImports(importScripts, rootDir);
+
+      var absoluteGlobs = fileGlobs.map(function (v) {
         return path.join(rootDir, v);
       });
+      var files = flatGlobs(absoluteGlobs);
+      var filesAndHashes = getFilesAndHashes(files, rootDir, absoluteGlobs);
+
+      var replacements = {
+        cacheId: cacheId,
+        ignoreUrlParametersMatching: config.ignoreUrlParametersMatching || [/./],
```

Comment:
Nit: I would remove this entirely, so no one is misled into thinking it has an effect.